app/models/article.rb

```diff
@@ -0,0 +1,4 @@
+class Article < ApplicationRecord
+  validates :title, presence: true, uniqueness: true
```

Comment:
What are the tradeoffs of enforcing uniqueness on article title? If we think about a site like Medium, do they enforce this? Or do they instead leverage a id-slug combination in their URL -- allowing people to create articles that are the same name since the uniqueness is provided from the id?